app/uk/gov/hmrc/transitmovements/controllers/MovementsController.scala

```diff
@@ -119,11 +121,16 @@ class MovementsController @Inject() (
   private def createEmptyMovement(eori: EORINumber, movementType: MovementType): Action[AnyContent] = Action.async(parse.anyContent) {
     implicit request =>
       val received = OffsetDateTime.ofInstant(clock.instant, ZoneOffset.UTC)
-      val movement = movementFactory.createEmptyMovement(eori, movementType, received, received)
+      val messageType = movementType match {
+        case MovementType.Arrival   => MessageType.ArrivalNotification
+        case MovementType.Departure => MessageType.DeclarationData
+      }
+      val message  = messageFactory.createEmptyMessage(messageType, OffsetDateTime.ofInstant(clock.instant, ZoneOffset.UTC), MessageStatus.Pending)
```

Comment:
Better to pass received to the empty message so that it has the same creation time as the movement and doesn't cause issues when polling

`messageFactory.createEmptyMessage(messageType, received, MessageStatus.Pending)`